### ISSUE TITLE
Keyboard shortcut for documenting a package

### ISSUE THREAD

**Issue by jennybc** [rstudio/rstudio#3699]
### System details

    RStudio Edition : Desktop
    RStudio Version : 1.2.1047
    OS Version      : macOS 10.14

### Describe the problem in detail

Discrepancies between various documents and conventions around keyboard shortcuts for documenting a package.

My definition of "document a package" = `devtools::document()`

Web document about shortcuts (https://support.rstudio.com/hc/en-us/articles/200711853-Keyboard-Shortcuts):

  * Says "Document Package" (good)
  * Recommends that a Mac user do `Cmd+Shift+D` (good & expected)

The first popup I get via *Tools → Keyboard Shortcuts Help*:

  * Says "Document" (good, I guess, but why not "Document Package"?)
  * Recommends ... `Ctrl + Shift + D` (surprising to a Mac user)

The bigger document I get via *Tools → Keyboard Shortcuts Help → See All Shortcuts* (file:///Applications/RStudio.app/Contents/Resources/www/docs/keyboard.htm):

  * Says "Document Package" (good)
  * Recommends that a Mac user do `Cmd+Shift+D` (good & expected)
  * Basically, it agrees with the version on the web.

What I see when I do *Tools → Modify Keyboard Shortcuts*:

  * Says "Roxygenize Package" (not good and not actually correct)
  * Recommends ... `Ctrl + Shift + D` (surprising to a Mac user)

**Comment by jmcphers** [comment]
@kevinushey addressed the major concerns in https://github.com/rstudio/rstudio/pull/3700. It does not address the Cmd/Ctrl inversion, though, which is a larger issue (across all shortcuts) than we are tracking with this bug.
